Local stuff for 64 Bit GCC plugin support etcetc

dreimer1986 opened the pull request:
+ Version hop to RosBE 2.3.0 sooner or later
+ Fix for ROSBE-171

dreimer1986 committed:
Bugfix for ROSBE-171

dreimer1986 committed:
Enable GMP as separate lib for the now enabled Plugin support of GCC. Needed for PSEH x64

dreimer1986 committed:
Merge branch 'reactos:master' into master

dreimer1986 committed:
Update ChangeLog.txt

Prepare new RosBE changelog.

dreimer1986 committed:
Version bump

dreimer1986 committed:
Fix the conflict panic reaction

dreimer1986 committed:
Merge branch 'reactos:master' into master

learn-more commented on RosBE-Windows/Root/ChangeLog.txt:
```diff
@@ -1,3 +1,13 @@
+*** Jun xxth, 2021 - RosBE 2.3.0 Released
```
2021?

dreimer1986 replied:
Now you know how long this stuff way rotting already. Will be fixed later.

HBelusca commented on RosBE-Windows/RosBE.nsi:
```diff
@@ -152,6 +152,7 @@ Section -BaseFiles SEC01
     SetOutPath "$INSTDIR\bin"
     SetOverwrite try
     File /r Components\bin\7z.exe
+	File /r Components\bin\7z.dll
```
```suggestion
    File /r Components\bin\7z.dll
```

HBelusca reviewed (commented):
What's the status of this PR?

dreimer1986 committed:
Merge branch 'reactos:master' into master

dreimer1986 committed:
Work to finally get to a new version.

Change all paths in NSIS to make creation of an installier more logical by just handing it a install root folder. Simplify the logic of the Shortcut creation to really work now.

Fix building of the platforn independent tools working on 64 Bit.

Dont hardcode the Host GCC to i386 anymore. This is needed to make a 64 Bit only setup for ROS 64 Bit.
